datacube_alchemist/_utils.py

```diff
@@ -142,6 +142,10 @@ def _munge_dataset_to_eo3(ds: Dataset) -> DatasetDoc:
     if ds.metadata_type.name == "eo":
         return _convert_eo(ds)
 
+    if ds.metadata_type.name == "eo_s2_nrt":
```

Comment:
Let's do this in the previous section with an `if ds.metadata_type.name in ["eo_s2_nrt", "eo_plus"]:`

Keep it simple!

Reply:
Done!